hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/CSMMetrics.java

```diff
@@ -216,6 +242,14 @@ public void incNumEvictedCacheCount() {
     numEvictedCacheCount.incr();
   }
 
+  public void incPendingApplyTransactions() {
+    pendingApplyTransactions.incr();
+  }
+
+  public void decPendingApplyTransactions() {
+    pendingApplyTransactions.incr(-1);
+  }
+
   public void unRegister() {
     MetricsSystem ms = DefaultMetricsSystem.instance();
     ms.unregisterSource(SOURCE_NAME);
```

Comment:
The `SOURCE_NAME` is different with `register` in the `create`, we should use a same name